src/mongocxx/test/client_side_encryption.cpp

```diff
@@ -49,6 +51,9 @@ const auto kLocalMasterKey =
     "\x4f\x69\x37\x36\x36\x4a\x7a\x58\x5a\x42\x64\x42\x64\x62\x64\x4d\x75\x72"
     "\x64\x6f\x6e\x4a\x31\x64";
 
+const auto kLocalKeyUUID = "\x2c\xe0\x80\x2c\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00";
```

Comment:
Can you comment noting this is a base64 decoding of `LOCALAAAAAAAAAAAAAAAAA==` and the value below a decoding of `AWSAAAAAAAAAAAAAAAAAAA==`? That will be easier to match up with the spec.

Reply:
Done.